spec/models/manageiq/providers/redhat/infra_manager/refresher_3_0_spec.rb

```diff
@@ -363,7 +363,7 @@ def assert_specific_vm_powered_on
       :ems_ref_obj   => nil,
       :uid_ems       => "eb5592d2-ce76-11e0-a703-005056af0085_vm",
       :name          => "vm",
-      :is_datacenter => false,
+      :type          => nil,
```

Comment:
I thought this was normal, experimentally it seems like the base class type is always nil... e.g.:
```
irb(main):006:0> VmOrTemplate.new.type
=> nil
irb(main):007:0> Vm.new.type
=> "Vm"
irb(main):008:0> Host.new.type
=> nil
irb(main):009:0> ManageIQ::Providers::Vmware::InfraManager::Host.new.type
=> "ManageIQ::Providers::Vmware::InfraManager::Host"
```